pilot/docker/Dockerfile.proxyv2

```diff
@@ -1,29 +1,16 @@
-# BASE_DISTRIBUTION is used to switch between the old base distribution and distroless base images
-ARG BASE_DISTRIBUTION=debug
-
-# Version is the base image version from the TLD Makefile
-ARG BASE_VERSION=latest
-
-# The following section is used as base image if BASE_DISTRIBUTION=debug
-FROM gcr.io/istio-release/base:${BASE_VERSION} as debug
-
-# The following section is used as base image if BASE_DISTRIBUTION=distroless
-# This image is a custom built debian11 distroless image with multiarchitecture support.
-# It is built on the base distroless image, with iptables binary and libraries added
-# The source can be found at https://github.com/istio/distroless/tree/iptables
-# This version is from commit 6faf2f9348d382c6a95bbc3171b89406103cb0ad.
-FROM gcr.io/istio-release/iptables@sha256:64ba191166d2c3f87815148cc1a9f530b770e0dfe81acb164ab79cbb31da582c as distroless
-
-# This will build the final image based on either debug or distroless from above
-# hadolint ignore=DL3006
-FROM ${BASE_DISTRIBUTION:-debug}
+FROM quay.io/centos/centos:stream8
 
 WORKDIR /
 
 ARG proxy_version
 ARG istio_version
 ARG SIDECAR=envoy
 
+# hadolint ignore=DL3041
+RUN dnf -y upgrade --refresh --nobest && \
+    dnf -y install iptables iproute openssl && \
```

Comment:
Do we need to install a specific versoin of openssl or just the latest one ?

Reply:
We need OpenSSL 1.1.1 and our istio-proxy expects to find libssl.so.1.1. In CentOS Stream 8 the package `openssl` is exactly this one that we expect. I don't think that this package will be replaced by another version, because the next one is 3.0.0 which provides breaking API and ABI and upgrading from 1.1.1 to 3.0.0 is a big challenge, so it will probably not happen.